lmctl/cli/commands/objectgroups.py

```diff
@@ -0,0 +1,50 @@
+import click
+from .utils import TNCOCommandBuilder, Identity, Identifier, pass_io
+from .actions import get, retry, rollback, cancel
+from lmctl.client import TNCOClient
+from lmctl.cli.format import Column
+from typing import List
+from lmctl.cli.io import IOController
+
+__all__ = (
+    'get_objectgroup',
+)
+
+from ..arguments import set_param_option
+
+tnco_builder = TNCOCommandBuilder(
+    singular='objectgroup',
+    plural='objectgroups',
+    display_name='Objectgroups'
```

Comment:
I think this will display better as `Object Groups`

Reply:
DONE